### PR TITLE
Fixes #9667

### LINKED ISSUE #9667: Prevent unsupported image types being selected in the wizard file selector
One of the theme reviewers noticed that [this theme](https://addons.mozilla.org/en-US/firefox/addon/whatsapp-mitlesen/) looks extremely pixelated in the "preview image" on AMO, but it looks fine when it is installed. 

Could we get the pixelation from preview to match how it appears upon installation? That would really help out the theme reviewers and end users.

### PR DESCRIPTION
**Pull request by miphilomath**
Changed the file type separator with comma(,) and added a function to verify the file upload type.

Signed-off-by: miphilomath <26297035+miphilomath@users.noreply.github.com>

* [x ] This PR relates to an existing open issue and there are no existing
      PRs open for the same issue.
* [x ] Add `Fixes #ISSUENUM` at the top of your PR.
* [x ] Add a description of the the changes introduced in this PR.

### CODE REVIEW

**Comment by miphilomath** [comment]
@eviljeff Please review :)

#9667

File: static/js/zamboni/static_theme.js [hunk 1]
```diff
@@ -11,7 +11,10 @@ $(document).ready(function() {
         function getFile() {
             file_selector = $wizard.find('#header-img')[0];
             file = file_selector.files[0];
+						if (!file.name.match(/.(jpg | jpeg | png | gif | apng)$/i))
```
Comment (eviljeff) [inline static/js/zamboni/static_theme.js]: This works great (apart from missing `.svg` off) but it'd be good to re-use the `accept` attribute from the html so the list of acceptable files is only defined in one place.  
How about:
`if (file && $wizard.find('#header-img').attr('accept').split(',').indexOf(file.type)== -1)` ?
It uses file.type to get content type which should hopefully be more robust too.

File: static/js/zamboni/static_theme.js [hunk 2]
```diff
@@ -11,7 +11,10 @@ $(document).ready(function() {
...
+							return null;
```
[the remark this answers is not in the document]
Comment (miphilomath) [reply static/js/zamboni/static_theme.js]: Ah! Must be my editor. Let me reformat these lines.

File: static/js/zamboni/static_theme.js (hunk 1 above)
Comment (miphilomath) [reply static/js/zamboni/static_theme.js]: Wow! That looks like a more clean and efficient way of doing the task. 
Thanks :)

**Comment by eviljeff** [comment]
r+ thanks for the contribution

**Comment by caitmuenster** [comment]
Thanks, @miphilomath, and welcome onboard! 🎉Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#November_2018). 

I look forward to seeing you around the project!

**Pull request merged**